### DISCUSSION TITLE
Social media card image issues

### DISCUSSION THREAD

**Discussion by secnate** [mmistakes/minimal-mistakes#2596] [Q&A] [answered]
## Environment

- Minimal Mistakes version:
- Ruby gem or remote theme version: Ruby Gem
- Jekyll version: I tried to determine what Jekyll version I am working with but the clue that I can provide as to the Jekyll version is that the website was started this week with the latest Jekyll type.
- Git repository URL: https://github.com/reshet-net/reshet-net.github.io
- GitHub Pages hosted (if yes provide URL to site): https://reshet-net.github.io/)
- Operating system: macOS catalina

## Expected behavior

I am trying to make the three twitter, facebook, and linkedin social media sharing buttons at the bottom of each blog post page share a link to a page on my website with a default image. I created a default og_image in the _config.yml file and labeled to an appropriate image.

Originally, I used this image, named *default_card.jpg*:
<img width="508" alt="Screen Shot 2020-07-09 at 10 30 54 PM" src="https://user-images.githubusercontent.com/33183110/87109946-e6d41200-c233-11ea-833c-ab2054ce65ba.png">

This is the old image.

I then got tired of this and went into the two assets folders in the repository and replaced it with the following image with the same *default_card.jpg* name and extension:
![default_card](https://user-images.githubusercontent.com/33183110/87110024-12ef9300-c234-11ea-8ae8-c4bf8aca7c71.jpg)

This is the only image present in the repository named default_card.jpg. Thus, I expected it to update everything and only the latest image preview being used due to the latest file overriding and replacing the older one.

Instead, I have the following two major issues:

- When I click the Twitter button, no image gets displayed at all. This is a consistent issue that I don't know how to remedy, given that I configured the _config.yml page properly in accordance with the Twitter Card requirements (https://mmistakes.github.io/minimal-mistakes/docs/layouts/#open-graph--twitter-card-images)

- When I go and click on the various sample blog posts and click the Facebook and LinkedIn share buttons below, I sometimes get the old image presented in the popover window, sometimes its the new one, and sometimes -- no image gets presented at all! 

Naturally, I want the newer image and only the newer image to get presented

## Steps to reproduce the behavior

I explained that I replaced the old image in the two folders with the newer image with the same name and extension. What's really weird is that I sometimes get the old one but sometimes the newer one. 

For example:

- https://reshet-net.github.io/blog/post-notice/ --> the linkedin share has no image, at least on my computer and the facebook share has the old image

- https://reshet-net.github.io/blog/post-quote/ --> the linkedin has the newer image but the facebook has the older image

- https://reshet-net.github.io/blog/welcome-to-jekyll/ --> the facebook and linkedin both have the old image

- https://reshet-net.github.io/blog/post-modified/ --> the linkedin has the newer image while the facebook has the older

I'd greatly appreciate your help. I don't know what to do and could use any advice you may be willing to give. Thanks for reading and in advance.

**Comment by mmistakes** [comment, marked as answer]
As far as I can tell you've done everything correct and I'm seeing your updated image. The social images are assigned properly in the HTML source for those pages.

The issue is on the various social platforms... they're caching the old image perhaps. Twitter and Facebook both have tools to inspect what meta data a page is using. The OpenGraph debugger tool also has a rescrape option to have your page updated.

https://cards-dev.twitter.com/validator

![Screen Shot 2020-07-10 at 7 23 45 AM](https://user-images.githubusercontent.com/1376749/87149488-540c9500-c27e-11ea-969c-7b49f2c6105b.png)

https://developers.facebook.com/tools/debug/

![Screen Shot 2020-07-10 at 7 24 42 AM](https://user-images.githubusercontent.com/1376749/87149550-76061780-c27e-11ea-9407-66423fb5bc3c.png)

Closing as this isn't a theme issue.

**Comment by michelmetran** [comment]
Maybe this can help
https://stackoverflow.com/questions/38376495/thumbnails-not-showing-up-on-linkedin-share-posts
